Index.html

```diff
@@ -145,108 +156,148 @@ <h3>Multi-Post Stories</h3>
+                    </div>
+                </div>
+                <div class="grid-item gridItem4">
+                    <div class="page page4">
+                        <img class="imgPlace" src="./images/ImgPlaceholder.png" alt="PlaceHolder">
+                        <img class="imgPlaceWide" src="./images/ImgPlaceholder_wide.png" alt="PlaceHolder">
+                        <div class="block">
+                            <h3>Multi-Post Stories</h3>
+                            <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the
+                                industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.
+                            </p>
+                            <ul>
+                                <li>
+                                    <a href="#">
+                                        <p>CSS</p>
+                                    </a>
+                                </li>
+                                <li>
+                                    <a href="#">
+                                        <p>html</p>
+                                    </a>
+                                </li>
+                                <li>
+                                    <a href="#">
+                                        <p>bootstrap</p>
+                                    </a>
+                                </li>
+                                <li>
+                                    <a href="#">
+                                        <p>Ruby</p>
+                                    </a>
+                                </li>
+                            </ul>
+                
+                            <button type="submit">See Project</button>
+                        </div>
                     </div>
                 </div>
             </div>
         </section>
         <section class="grid-item formContent">
-            <h5>CONTACT US</h5>
+            <h5 class="h5Contact">CONTACT US</h5>
+            <h5 class="h5Lets">Let's chat!</h5>
             <p>If you have an application you are interested in developing, a feature
                 that you need built or a project that needs coding. I'd love to help with it </p>
             <form action="https://formspree.io/f/mrgjqdpe" method="post">
                     <input class="textInput" type="text" name="name" placeholder="Enter your name" maxlength="30" required>
                     <input class="emailInput" type="email" name="email" placeholder="Enter your email" required>
                     <textarea class="area" name="comment" rows="7" cols="25" maxlength="500" placeholder="Enter your text here" required></textarea>
-                  <button type="submit">See Project</button>
+                  <button class="submitButton" type="submit"></button>
             </form>
         </section>
     </div>
     <footer>
```

Comment:
Figma Design          |  Your design     
:-------------------------:|:-------------------------:
![bbb](https://user-images.githubusercontent.com/69638013/155659545-832247c5-8baa-4c42-89a4-b1af0d483d44.PNG)  | ![aaaa](https://user-images.githubusercontent.com/69638013/155659558-ff61237e-b0a8-49e2-a112-4198c0d7528e.PNG)

- [ ] Kindly align footer icons to the center just like the figma design 👍🏿